library/game_state.py

```diff
@@ -0,0 +1,167 @@
+import time
+
+import blessed
+
+from library.user_term import convert_to_space_location
+
+from .board import Board
+from .user_term import update_user_section
+
+
+class GameState:
+    """Main code for managing the game state"""
+
+    def __init__(self, term: blessed.Terminal):
+        self.current: int = 0
+        self.good_move = False
+        self.save_subgrid_bool: bool = False
+        self.update_board: bool = False
+        self.player_active: int = 0
+        self.term_info: list[str] = [""] * 3
+        self.user_select_subgrid: int = 0
+        self.user_select_space: int = 0
+        self.user_input: str = ""
+
+        self.term_info[1] = "shall we play a game?"
+        self.term_info[2] = "(y/n?)"
+        self.redraw_user_term(term)
+        self.next: int = 10
+
+    def driver(self, term: blessed.Terminal, board: Board) -> None:
+        """Main State Machine"""
+        if self.next == 10:

```

Comment:
We can use an `enum.IntEnum` for `self.next`.

```py
class NextPossibilities(enum.IntEnum):
    WAIT_FOR_READY = 10
    ...
```
then just `self.next == NextPossibilities.WAIT_FOR_READY`

Reply:
I will look into this on the next go around? Will this hold up the merge?

Reply:
We can do it later, sure.